play-scala-tls-example/scripts/test-sbt

```diff
@@ -5,4 +5,10 @@ export KEY_PASSWORD=`cat scripts/password`
 echo "+---------------------------------------+"
 echo "| Executing tests using (modified sbt)  |"
 echo "+---------------------------------------+"
+
+# Necessary to first download the alpn agent jar.
+sbt stage
```

Comment:
Shouldn't be necessary with https://github.com/playframework/play-samples/blob/05a73aaba3cc404677ca4fa67a9d1d4ec6a83621/play-scala-tls-example/play#L64-L73

Reply:
rolled-back